flashinfer/sampling.py

```diff
@@ -806,6 +838,66 @@ def top_k_sampling_from_probs(
     )
 
 
+def radik_sampling_from_probs(
```

Comment:
I would also encourage dispatching topk sampling to this function for small K's.